tests/schema_update_test.go

```diff
@@ -0,0 +1,69 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/bytebase/bytebase/api"
+	"github.com/bytebase/bytebase/plugin/db"
+)
+
+func TestSchemaUpdate(t *testing.T) {
```

Comment:
OK for the first version.

But I feel this is still hard to write. I am thinking of defining a test case like a script, so one can write

Login
// validate yyy
GET url
// validate xxx
POST url

And we have a test driver to parse that script and execute the case. Likely there is already people does the same thing and we can do some research and come up with our solution. wdyt?

Reply:
I might prefer to use Go script for testing. The HTTP messaging part isn't quite interesting, but the validation could be complex that really needs coding, especially if we want to check some side-effects such as activity.

Reply:
Let's write couple tests and see if we can generalize the pattern